sys/ansible/playbooks/deploy.yml

```diff
@@ -99,12 +99,20 @@
       notify:
       - reload nginx
 
-    - name: "Reset opcache"
-      shell: |
-        curl -sO https://gordalina.github.io/cachetool/downloads/cachetool.phar
-        chmod +x cachetool.phar
-        php cachetool.phar opcache:reset --fcgi
-        rm cachetool.phar
+    - name: "Check cachetool.phar already exists"
+      stat: path="{{deploy_helper.shared_path}}/cachetool.phar"
+      register: stat_cachetool_result
+
+    - name: "Download cachetool.phar"
+      get_url:
+        url: "https://gordalina.github.io/cachetool/downloads/cachetool-3.2.1.phar"
```

Comment:
Sorry, my mistake, replace url with `https://gordalina.github.io/cachetool/downloads/cachetool-5.0.0.phar`  
The version `3.2.1` it's for php70!

Reply:
fixed